susemanager-utils/susemanager-sls/salt/bootstrap/remove_traditional_stack.sls

```diff
@@ -46,6 +46,7 @@ remove_traditional_stack:
       - module: disable_repo*
 {%- endif %}
     - unless: rpm -q spacewalk-proxy-common
+    - unless: rpm -q patterns-uyuni_server
```

Comment:
I would not check for the pattern as the name of the pattern might be different in "SUSE Manager".
Just check for `spacewalk-common`. This is a package only for the server. In spacewalk it acted like a pattern and has just dependencies to server packages.

Reply:
Fixed, thanks!